model/docs/extending_pyrenew.qmd

```diff
@@ -0,0 +1,256 @@
+---
+title: Extending pyrenew
+format: gfm
+---
+
+This tutorial illustrates how to extend `pyrenew` with custom `RandomVariable` classes. We will use the `InfectionsWithFeedback` class as an example. The `InfectionsWithFeedback` class is a `RandomVariable` that models the number of infections at time $t$ as a function of the number of infections at time $t - \tau$ and the reproduction number at time $t$. The reproduction number at time $t$ is a function of the *unadjusted* reproduction number at time $t - \tau$ and the number of infections at time $t - \tau$:
+
+$$
+\begin{align*}
+I(t) & = \mathcal{R}(t)\sum_{\tau=1}^{T_g}I(t - \tau)g(\tau) \\
+\mathcal{R}(t) & = \mathcal{R}^u(t)\exp\left(-\gamma(t)\sum_{\tau=1}^{T_f}I(t - \tau)f(\tau)\right)
+\end{align*}
+$$
+
+Where $\mathcal{R}^u(t)$ is the unadjusted reproduction number, $g(t)$ is the generation interval, $\gamma(t)$ is the infection feedback strength, and $f(t)$ is the infection feedback pmf.
+
+## The expected outcome
+
+Before we start, let's simulate the model with the original `InfectionsWithFeedback` class. We will simulate the model with no observation process. The following code-chunk loads the required libraries and defines the model components:
```

Comment:
Why do we simulate the model with no observation process?

Reply:
To make it simpler. We can always use the NB process if you prefer.

Reply:
Please add some explanation in the qmd, so the reader does not ask the same question

Reply:
Sounds good!

Reply:
Addressed in b603eb5